src/main/webapp/app/scores/exercise-scores.component.html

```diff
@@ -173,21 +177,24 @@ <h2>
                     <ng-template ngx-datatable-cell-template let-value="value"> {{ value }} min</ng-template>
                 </ngx-datatable-column>
 
-                <ngx-datatable-column prop="" *ngIf="exercise.type === PROGRAMMING" [resizeable]="false" [minWidth]="450" [width]="450">
+                <ngx-datatable-column prop="" *ngIf="exercise.type === ExerciseType.PROGRAMMING" [resizeable]="false" [minWidth]="450" [width]="450">
                     <ng-template ngx-datatable-cell-template let-value="value">
-                        <button *ngIf="exercise.type === PROGRAMMING" class="btn btn-primary btn-sm mr-1" (click)="goToRepository(value)">
-                            <fa-icon [icon]="'code-branch'" class="mr-1"></fa-icon>Repository
+                        <button *ngIf="exercise.type === ExerciseType.PROGRAMMING" class="btn btn-primary btn-sm mr-1" (click)="goToRepository(value)">
```

Comment:
I think the user's main wish was that he can quickly get from the result to the submission content itself (file, diagram, text, etc). But since this leads to the assessment page for most exercise types, we should probably also link to the assessment page for a file upload exercise (and as you said, there will probably be users who appreciate the possibility to access the assessment page and not just the file itself). It will be one more click to see the file content itself but that should be tolerable.

Reply:
Implemented the change for the file upload exercise.